QuizApp/User/UserClient.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+protocol UserClientProtocol {
+
+    func getUser(accessToken: String) async throws -> UserResponseModel
+
+}
+
+class UserClient: UserClientProtocol {
+
+    let baseURL = "https://five-ios-quiz-app.herokuapp.com/"
+    let userPath = "api/v1/account"
+
+    private let apiClient = ApiClient()
+
+    func getUser(accessToken: String) async throws -> UserResponseModel {
+        guard let URL = URL(string: "\(baseURL)\(userPath)") else {
+            throw RequestError.invalidURL
+        }
+
+        var URLrequest = URLRequest(url: URL)
+        URLrequest.addValue("Bearer \(accessToken)", forHTTPHeaderField: "Authorization")
+        URLrequest.httpMethod = "GET"
```

Comment:
You have the same logic in [LoginClient](https://github.com/barbarakos/QuizApp/pull/8/files#diff-0843ad781464f144baff945c23410e5d9b32cd9a7ad87b4bbf8a1ee0e8a6b541R33). You can generalize base api client to gave a generic `get` function.